source/_integrations/tesla.markdown

```diff
@@ -28,6 +28,8 @@ ha_platforms:
 
 The `Tesla` integration offers integration with the [Tesla](https://auth.tesla.com/login) cloud service and provides presence detection as well as sensors such as charger state and temperature.
 
+Note: MFA on your Tesla Account is not supported at this time.
```

Comment:
- I think this should be in the configuration section (not in the introduction text).
- Format this as a note, see: <https://developers.home-assistant.io/docs/documenting/create-page#html>

Reply:
I've changed the suggested edits.